srcpkgs/libguestfs-tools/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'libguestfs-tools'
+pkgname=libguestfs-tools
+_pkg=guestfs-tools
+version=1.48.0
+revision=1
+_version_short=${version%.*}
+wrksrc="${_pkg}-${version}"
+build_style=gnu-configure
+configure_args="ac_cv_prog_NCURSES_CONFIG=ncursesw6-config"
+hostmakedepends="flex pkg-config ncurses-devel gettext-devel pcre2-devel libvirt-devel
+ libxml2-devel jansson-devel ncurses-devel ocaml ocaml-findlib perl perl-Locale-TextDomain
+ hivex perl-Module-Build bash-completion qemu cpio"
+makedepends="cdrtools xz sqlite libguestfs-devel"
+depends="cdrtools xz sqlite"
+checkdepends="libguestfs libguestfs-devel curl wget tar procps-ng"
+short_desc="Additional tools for virtual machine disk image manipulation"
+maintainer="J Farkas <chexum+git@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/rwmjones/guestfs-tools"
+#changelog=""
+distfiles="https://download.libguestfs.org/${_pkg}/${_version_short}-stable/${_pkg}-${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://download.libguestfs.org/guestfs-tools/${version%.*}-stable/${_pkg}-${version}.tar.gz"
```

Reply:
_version_short is indeed just overengineering, however _pkg (if the package is meant to have a different name), is needed in more places.  I have pushed the amended commit.

Reply:
The url and wrksrc will always be what upstream uses, there is no point in using `$pkgname` or `$_pkg` in it, the only variable that make sense is `$version` since that is supposed to change.